extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -303,6 +303,17 @@ public function getWikiVariables() {
 		$this->response->setCacheValidity( self:: WIKI_VARIABLES_CACHE_TTL );
 	}
 
+	private function getRedirectTitle() {
+		$title = $this->getTitleFromRequest();
+
+		if ( $title->isRedirect() ) {
+			$article = Article::newFromID( $title->getArticleId() );
```

Comment:
Maybe it's not needed to have this local var here? Just return Article::newFromID( $title->getArticleId() )->getRedirectTarget() :)